### ISSUE TITLE
MSH 4.1 breaks some examples

### ISSUE THREAD

**Issue by gdmcbain** [kinnala/scikit-fem#129]
[gmsh#444 MSH 4.x format revisions](https://gitlab.onelab.info/gmsh/gmsh/issues/444) flagged some changes to the format output by Gmsh.  To see whether this affects us here, I ran, as in #9,
```shell
parallel python {} ::: docs/examples/*.py
```
with a very freshly updated and compiled Gmsh 4.2.0-git-d474ec1e2 on 64-bit PureOS GNU/Linux 8.

This raised:
* `ValueError` in ex13 and backward_facing_step
* `OverflowError` in ex12, ex17, and ex20.

The test suite is passing
```shell
pytest -n auto tests
```
so I expect that this issue only involves examples making use of optional external packages.

I'll investigate this shortly.

**Comment by gdmcbain** [comment]
Suspecting `pygmsh.generate_mesh`, used by all five failing examples, I raised pygmsh#240 upstream.

**Comment by gdmcbain** [comment]
Upstream fix accepted in nschloe/pygmsh#242 and proposed in nschloe/meshio#331.  Using branch [msh4-1-330](https://github.com/gdmcbain/meshio/tree/msh4-1-330) on gdmcbain/meshio, all five broken examples pass; indeed, all examples and tests pass.

**Comment by kinnala** [comment]
Yeah, I'd love to have all the examples tested. I've been too busy to focus on this. I plan to finish adding everything to docs before I work on the tests.

**Comment by gdmcbain** [comment]
> Yeah, I'd love to have all the examples tested. 

This is a really good idea. I suppose we'll want to be able to elegantly skip examples requiring uninstalled optional dependencies. I think pytest has something for that, used to make VTK, XML, HDF5, &c., optional in meshio. It might be possible to concoct something similar for unittest.

**Comment by gdmcbain** [comment]
> Can I close this?

Well it's still broken with the latest Gmsh and meshio (pending acceptance of nschloe/meshio#331), but I don't think that there's any work to be done here.

**Comment by gdmcbain** [comment]
Following the merger of nschloe/meshio#331, all five examples relying on pygmsh work again, but for the moment only with git-pulled master, not pip-installed meshio, pending a version bump.

**Comment by gdmcbain** [comment]
Fixed by nschloe/meshio#367.
